DigitalOffice/Services/CompanyService/Validators/EditPositionRequestValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.CompanyService.RestRequests;
+
+namespace LT.DigitalOffice.CompanyService.Validators
+{
+    public class EditPositionRequestValidator : AbstractValidator<EditPositionRequest>
+    {
+        public EditPositionRequestValidator()
+        {
+            RuleFor(position => position.Id)
+                .NotEmpty()
+                .WithMessage("Empty position id.");
+
+            RuleFor(position => position.Name)
+                    .NotEmpty()
+                    .WithMessage("Position name is empty.")
+                    .MaximumLength(80)
```

Comment:
Hm...
"Inspector of the department for work with personnel of the Directorate for the management of institutions with special conditions of economic activity of the Directorate of the Federal Penitentiary Service of Russia in the Kirov region" (236)

Reply:
This is a joke, but I would leave a comment here that I'm not sure.

Reply:
Ahahah, nice joke, dude
But I think these details should be in Description, shouldn't they?